### DISCUSSION TITLE
Where does AudioGridderPluginTray.exe go when manually installed?

### DISCUSSION THREAD

**Discussion by rockumk** [apohl79/audiogridder#904] [Help]
I am trying to give my students the ability to connect to my AudioGridder server with their student laptops. However, student machines have no admin or install rights.  I can get the plugins to the student machines, but when the VST plugin is loaded by the DAW host it complains that it can not find "AudioGridderPluginTray.exe".  Where should I put this executable so that the plugin can find and run it correctly?  Can it be installed without admin rights?

**Comment by apohl79** [comment]
It goes to C:/Program Files/AudioGridderPluginTray/AudioGridderPluginTray.exe

I guess there is no way w/o admin rights. You could disable the tray app in the plugin.

**Reply by rockumk** [reply]
OK thanks, if you can ever make it a portable install it would be nice, but I realize it's probably not a feature that is in high demand.
